doc/src/user_guide.rst

```diff
@@ -592,10 +617,11 @@ Example::
     quad-deg = 10
     quad-pts = gauss-legendre
 
-[solver-interfaces-tri]
-^^^^^^^^^^^^^^^^^^^^^^^
+[solver-interfaces-tri-{mg-*p*}]
```

Comment:
I did not do a particularly good job explaining what I meant.  An example of a config line is [solver-interfaces-tri-mg-p3].  Hence why it is important to indicate that the p is required, rather than something which is substituted.